template-parts/entry-header.php

```diff
@@ -31,8 +31,27 @@
 			<?php
 		}
 
-		// Default to displaying the post meta.
-		twentytwenty_the_post_meta( get_the_ID(), 'single-top' );
+		// On pages with the cover template, display a "To the content" link.
```

Comment:
The cover template does not use entry-header.php, the related code is now in https://github.com/WordPress/twentytwenty/blob/master/template-parts/content-cover.php#L56

Reply:
@carolinan I resolved that conflict in https://github.com/WordPress/twentytwenty/pull/251/commits/b5df3b262fd82ecfe1a8670fa115852fc97b96c9